docs/quality-assurance-test-cases.md

```diff
@@ -173,7 +173,29 @@ This document contains some examples of test cases for each feature implemented
 | Logged in user tries to complete a task from an non existing request (The request  exists in a different relationship) | Fail |
 | Logged in user tries to complete a task from a request which is not in the ACCEPTED state (as a mentor or as a mentee) | Fail |
 
+### List All Tasks
+**Service:** PUT /mentorship_relation/{request_id}/task/{task_id}
 
+| Test Case  | Outcome |
+| ---------- | -----------|
+| Logged in user requests list all task,or out of all task, those which involves the user (as a mentor or as a mentee) | Success |
+| Logged in user requests list all task,or out of all task, those which involves the user (neither as a mentor nor as a mentee) | Fail |
+| Logged in user requests to list all task whose id does not exist, which involves the user (as a mentor or as a mentee) | Fail |     
```

Comment:
As there will be no `task_id` the test cases will resolve to following cases : 
1. Logged in user requests all tasks for a mentorship relation in which user is involved | Success
2. Logged in user requests all tasks for a mentorship relation in which user isn't involved | Fail
3. Logged in user requests all tasks for a invalid mentorship relation | Fail
4. Not logged in user requests all tasks for any mentorship relation | Fail
5.  ...

These are just rough approximation of the cases, with rough language.  
If the cases aren't clear, you can refer the backend function and list the cases accordingly.

Reply:
sorry, i am unable to find the backend function. can to please give me the link to refer.